docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md

```diff
@@ -1,30 +1,43 @@
 ---
-title: Debug add-ins using F12 developer tools on Windows 10
+title: Debug add-ins using developer tools on Windows 10
 description: ''
-ms.date: 03/19/2019
+ms.date: 07/01/2019
 localization_priority: Priority
 ---
 
-# Debug add-ins using F12 developer tools on Windows 10
+# Debug add-ins using developer tools on Windows 10
+
+If you are not using an IDE like Visual Studio, or if you need to investigate a problem while running your add-in outside the IDE, there are developer tools available to help you debug the add-in on Windows 10. 
```

Comment:
Could this be rephrased to not start the article with a conditional? Something like "There are developer tools outside of IDEs available to help you debug your add-ins on Windows 10. These are useful when you need to investigate a problem while running your add-in outside the IDE."